src/views/NutritionView.vue

```diff
@@ -297,6 +301,13 @@ export default {
           ];
       }
     },
+    
+    getNutritionValue: async function(){
+      const newProducts = await getProducts(10);  
```

Comment:
To be sure you understood, this line of code asks for the page 10 of the procts having a picture with nutrition but without nutrition information

Reply:
Hi @alexfauquette

I have constructed the OCR URL

> To be sure you understood, this line of code asks for the page 10 of the procts having a picture with nutrition but without nutrition information

I wanted to ask how to verify if the data I am getting is correct and from the right page. 

Because one code has multiple `nutrition` keys of different languages.

Reply:
Not sure the different language is the biggest problem here.

Currently what is doing the app:
- You have a buffer with products which is filled by the method `addProducts`
- You take the first product of this buffer (For example some chocolate), to show its picture, and ask to fill the nutritional table
- Now in this part of your code, you fetch 10 random products (for example pizza) and extract the nutritional information

So at the end you will put the estimated nutrition facts of a pizza inside table of a chocolate

About this problem, `newProducts[0]` should not be a new product. It should be the product currently displayed. Which you can pass as an argument of this method

Reply:
From a larger perspective, I feel like this issue is too complicated compared to your level in vueJS. 

I think you will learn much more by starting with easier issues, or with someone to guide you more precisely than what I can do